app/dose-event-service/dose-event.mocks.ts

```diff
@@ -1,39 +1,20 @@
 import {DoseEvent} from "../dose-event/dose-event";
-
-const now: Date = new Date();
-const fifteenMinAgo: Date = new Date(now.getTime() - 15 * 60000);
-const fortyFiveMinAgo: Date = new Date(now.getTime() - 45 * 60000);
-const fifteenMinFromNow: Date = new Date(now.getTime() + 15 * 60000);
+import {DOSE_AMIGOS_USERS} from "../dose-amigos-user-service/dose-amigos-user-mocks";
 
 export var DOSE_EVENTS: Array<DoseEvent> = [
     {
         id: 3,
-        time: now,
-        user: {
-            id: 300,
-            name: "Barney Rubble",
-            lastTimeDoseTaken: now,
-            nextTimeDoseScheduled: fifteenMinFromNow
-        }
+        time: DOSE_AMIGOS_USERS[0].lastTimeDoseTaken,
+        user: DOSE_AMIGOS_USERS[0]
```

Comment:
Updated this mock DoseEvents object to use the mock DoseAmigosUsers for its users.